packages/strapi-helper-plugin/lib/src/components/InjectionZone/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useStrapi from '../../hooks/useStrapi';
+
+const InjectionZone = ({ area, ...props }) => {
+  const { strapi: globalStrapi } = useStrapi();
+
+  const [plugin, page, position] = area.split('.');
+  const cmPlugin = globalStrapi.getPlugin(plugin);
```

Comment:
`const plugin` it won't always be the content manager

Reply:
can you add this statement:

```
if (!plugin) {
  return null;
}
```

Reply:
Woops, you're right 😅